testing/tests/DevExpress.ui.widgets/popup.tests.js

```diff
@@ -1453,6 +1478,23 @@ QUnit.module('templates', () => {
         assert.equal($popupContent.find(toSelector('changed-test-title-renderer')).length, 1, 'option \'titleTemplate\' successfully passed to the popup widget');
     });
 
+    QUnit.test('titleRendered event should be fired if was set thought method', function(assert) {
+        assert.expect(1);
+
+        const $element = $('#popup').dxPopup({
+            visible: true,
+            showTitle: true
+        });
+        const instance = $element.dxPopup('instance');
+
+        instance.on('titleRendered', function(e) {
```

Comment:
sinon?

Reply:
There is no need to use it. I just make an assertion in this handler.

Reply:
Ok